### DISCUSSION TITLE
filename symbols => newfile

### DISCUSSION THREAD

**Discussion by mezdelex** [nvim-lualine/lualine.nvim#1083] [Q&A]
When should the newfile symbol appear?

I'm trying to reproduce an scenario where newfile symbol will appear, either creating new files via NvimTree and also using the [No Name] buffer when opening the editor with 'nvim' command alone. In the second case, after giving it a name with :w <name> and saving, the command line below shows [New] but lualine is not showing the [New] symbol accordingly. Is that symbol not necessary? In which scenario would it be used?

Basically I want to know if that 'newfile' symbol of 'filename' section would be shown alone like the [No Name] symbol or after an actual name like when modifying files or locked files, to either leave it as [New] or change it for an icon respectively assuming that it's working properly.

Thanks in advance.

EDIT: Nevermind, I found the answer, it's working as intended obviously.
